plansys2_domain_expert/src/plansys2_domain_expert/DomainExpertNode.cpp

```diff
@@ -114,6 +114,7 @@ DomainExpertNode::on_configure(const rclcpp_lifecycle::State & state)
     std::istreambuf_iterator<char>());
 
   auto planner = std::make_shared<plansys2::POPFPlanSolver>();
+  planner->configure(shared_from_this(), "POPF");
```

Comment:
Caught this bug with these changes -- we were relying on an unconfigured solver node here.

Reply:
good catch :+1: